katsdpingest/test/test_ingest_server.py

```diff
@@ -232,6 +239,9 @@ async def setUp(self, context, command_queue):
         self._telstate['sdp_model_base_url'] = 'http://test.invalid/'
         rfi_mask_model_key = self._telstate.join('model', 'rfi_mask', 'fixed')
         self._telstate[rfi_mask_model_key] = 'rfi_mask/fixed/sha256_deadbeef.hdf5'
+        band_mask_model_key = self._telstate.join(
+            'i0_antenna_channelised_voltage', 'model', 'band_mask', 'fixed')
+        self._telstate[band_mask_model_key] = 'band_mask/fixed/sha256_cafebeef.hdf5'
```

Comment:
What is cafebeef all about ?

Reply:
Maybe https://www.artima.com/insidejvm/whyCAFEBABE.html will shed some light?

Reply:
Sure :) But isn't the mock side_effect returning the model directly for fetcher.get() ? I don't see how the URL is actually used in this test ?

Reply:
True, and I should probably be less lazy and make the side_effect a function that will check the URL. It still needs *some* value in telstate for the code that constructs the URL. I'll revisit after I've moved some of the `_load_model` functionality into katsdpmodels.

Reply:
Done.